prompt2model/param_selector/search_with_optuna.py

```diff
@@ -0,0 +1,245 @@
+"""This module provides automatic hyperparameter selection using optuna."""
```

Comment:
Capitalize Optuna.

```suggestion
"""This module provides automatic hyperparameter selection using Optuna."""
```

Reply:
Please address this comment before merging!

Reply:
Done